src/core/server/plugins/plugins_system.test.ts

```diff
@@ -117,15 +117,17 @@ test('`setupPlugins` throws if plugins have circular optional dependency', async
 
 test('`setupPlugins` ignores missing optional dependency', async () => {
   const plugin = createPlugin('some-id', { optional: ['missing-dep'] });
-  jest.spyOn(plugin, 'setup').mockResolvedValue('test');
+  jest.spyOn(plugin, 'setup').mockResolvedValue({ foo: 'bar' });
```

Comment:
TypeScript found these "error" in tests&mdash;plugin contract must be an object.